src/test/rustdoc/macro-private-not-documented.rs

```diff
@@ -0,0 +1,19 @@
+// Checks that private macros aren't documented by default. They
+// should be still be documented in `--document-private-items` mode,
+// but that's tested in `macro-document-private.rs`.
```

Comment:
nit: Can you combine this file with the test you wrote in the other PR? Rustdoc tests are really slow to run and I haven't gotten around to investigating it yet.

Reply:
Which one? This fits best logically with `macro-document-private.rs`, but that needs `--document-private-items` and this one needs not to have that option. The only rustdoc test I added that doesn't have that option is `macro-indirect-use.rs`, but this doesn't fit very well there.

Reply:
Ah ok, fair enough. No need to worry about it.